neuralmonkey/decoders/beam_search_decoder.py

```diff
@@ -141,10 +146,14 @@ def get_initial_loop_state(self) -> BeamSearchLoopState:
 
         # We want to feed these values in ensembles
         self._search_state = SearchState(
-            logprob_sum=tf.placeholder_with_default([0.0], [None]),
+            input_beam_size=tf.placeholder_with_default(
+                1, None, name="input_beam_size"),
```

Comment:
tady ten shape má bejt spíš `[]`, ne? ne `None`..

Reply:
taky když je to na dalším řádku tak bych tam napsal ty názvy těch argumetů jako `shapes=`... ale jen jestli se to vejde

Reply:
Ok, nebyl jsem si jisty se syntaxi.